src/routes/web.php

```diff
@@ -26,3 +27,5 @@
 Route::view('/home', 'home')->middleware('auth');
 
 Route::get('/logout', LogoutController::class);
+
+Route::get('/homeController', HomeController::class)->name('homeController');
```

Comment:
¿/homeController? Remove the line 27, and add to this route the middleware auth, in other to prevent that not logged user can access to this page. And rename /homeController to /.